web/src/routes/RepositoriesList/RobotAccountsList.tsx

```diff
@@ -478,6 +511,9 @@ export default function RobotAccountsList(props: RobotAccountsListProps) {
   }
   return (
     <>
+      <AlertGroup isToast isLiveRegion>
```

Comment:
Do we want to use the `useAlerts` hook here instead of creating them directly in the component?